### PR TITLE
update TOC members: add psschwei

### PR DESCRIPTION
**Pull request by psschwei**
Signed-off-by: Paul S. Schweigert <paul@paulschweigert.com>

Filling Scott's seat, xref #1167

### CODE REVIEW

File: TECH-OVERSIGHT-COMMITTEE.md [hunk 1]
```diff
@@ -113,7 +114,7 @@ To recognize the folks that have served in the TOC in the past, below we list th
 | <img width="30px" src="https://github.com/julz.png">           | Julian Friedman | [@julz](https://github.com/julz)                     | 2021-05-26 | 2022-02-07 |
 | <img width="30px" src="https://github.com/rhuss.png">          | Roland Huß      | [@rhuss](https://github.com/rhuss)                   | 2021-02-16 | 2022       |
 | <img width="30px" src="https://github.com/mattmoor.png">       | Matt Moore      | [@mattmoor](https://github.com/mattmoor)             | 2022-02-07 | 2022       |
-| <img width="30px" src="https://github.com/n3wscott.png">       | Scott Nichols   | [@n3wscott](https://github.com/n3wscott)             | 2022-01-11 | 2022-09-30 |
+| <img width="30px" src="https://github.com/n3wscott.png">       | Scott Nichols   | [@n3wscott](https://github.com/n3wscott)             | 2022-01-11 | 2022-09-27 |
```
Comment (csantanapr) [inline TECH-OVERSIGHT-COMMITTEE.md:117]: LOL!

**Comment by psschwei** [comment]
For the groups update
/assign @kvmware @upodroid

**Review by upodroid — APPROVED** [review]

**Comment by csantanapr** [comment]
thank you @upodroid

**Pull request merged**